pkgs/blast_repo/bin/blast_repo.dart

```diff
@@ -31,11 +32,23 @@ Future<void> main(List<String> args) async {
       help: 'Prints out usage and exits',
     );
 
-  final argResults = parser.parse(args);
-
-  if (argResults['help'] as bool) {
+  void printUsage() {
     print('Usage: $packageName <options> [org/repo]\n');
     print(parser.usage);
+  }
+
+  final ArgResults argResults;
+  try {
+    argResults = parser.parse(args);
```

Comment:
I feel that `package:args` does a really good job w/ multi-command tools, but that for single command tools you need to have a bunch of boilerplate to catch errors, print usage, print help, ...

Reply:
agreed!